tests/Event/Commands/Moderation/PublishTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Commands\Moderation;
+
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use PHPUnit\Framework\TestCase;
+
+final class PublishTest extends TestCase
+{
+    /**
+     * @test
```

Comment:
Maybe reference that this is a `@bugfix` and add a `@see` link to the ticket so we don't forget why this was overridden later.

Reply:
<img width="1058" alt="Screenshot 2022-05-17 at 15 36 23" src="https://user-images.githubusercontent.com/1322949/168823850-60e4337e-0c06-4634-af42-8909a748c51e.png">

Reply:
Sure! 👍